src/main/scala/viper/gobra/translator/Translator.scala

```diff
@@ -37,8 +39,36 @@ object Translator {
       case (t, transformer) => transformer.transform(t)
     }
 
-    config.reporter report GeneratedViperMessage(config.taskName, config.packageInfoInputMap(pkgInfo).map(_.name), () => transformedTask.program, () => transformedTask.backtrack)
+    config.reporter report GeneratedViperMessage(config.taskName, config.packageInfoInputMap(pkgInfo).map(_.name), () => sortAst(transformedTask.program), () => transformedTask.backtrack)
     transformedTask
   }
 
+  def sortAst(program: vpr.Program): vpr.Program = {
```

Comment:
I would add a small comment here explaining that we are doing this to make the generated code easier to navigate